### PR TITLE
CBI-415: Prepare Github workflow to push PoS Network Node image to public registry

### PR DESCRIPTION
**Pull request by andrii-maglovanyi**
chore: add publishing to DockerHub with package name and version, taken from `Cargo.toml` file

### CODE REVIEW

File: .github/workflows/publish-image-dockerhub.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Publish image to DockerHub
+on:
+  push:
+    branches:
+      - cere-master
```
Comment (AndreiNavoichyk) [inline .github/workflows/publish-image-dockerhub.yml]: Agreed to use single branch and tags as a trigger for publishing to public registry.

File: .github/workflows/build-and-push-docker-image.yml [hunk 2]
```diff
@@ -1,32 +1,32 @@
-name: 'Build and push Docker image'
+name: Build and push image to ECR
 on:
   push:
     branches:
-      - 'dev-cere'
+      - cere-master
```
Comment (AndreiNavoichyk) [inline .github/workflows/build-and-push-docker-image.yml]: Rename to `master-cere`

File: .github/workflows/build-and-push-docker-image.yml [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
+      
 env:
-  ECR_REPOSITORY: 'crb-substrate-testnet-node'
-  IMAGE_TAG: ${{ github.run_number }}-${{ github.sha }}
+  ECR_REPOSITORY: crb-substrate-testnet-node
```
Comment (AndreiNavoichyk) [inline .github/workflows/build-and-push-docker-image.yml]: We need to create a repo `pos-network-node` in private registry. Please, work with #devops team to do it.

File: .github/workflows/publish-image-to-dockerhub.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+          docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $DOCKERHUB_REPOSITORY/$PACKAGE_NAME:${GITHUB_REF##*/}
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Publish image to DockerHub
+        id: docker_publish
+        run: |
+          docker push $DOCKERHUB_REPOSITORY/$PACKAGE_NAME:${GITHUB_REF##*/}
```
Comment (AndreiNavoichyk) [inline .github/workflows/publish-image-to-dockerhub.yml:43]: Please, update `latest` tag with newest version.

File: .github/workflows/build-and-push-docker-image.yml (hunk 3 above)
Comment (andrii-maglovanyi) [reply .github/workflows/build-and-push-docker-image.yml]: Requested devops team https://www.notion.so/cere/Create-new-private-ECR-repository-5b3e95b9023a4f3d89459382e5cc62dc

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**